tools/batchL2DataCreatedRollup/.gitignore

```diff
@@ -0,0 +1,2 @@
+input.json
+info.json
```

Comment:
We can in fact delete this file in the second script since it is just intermediate information